spec/classes/foreman_config_spec.rb

```diff
@@ -84,7 +84,7 @@
 
       it 'should contain foreman::config::passenger' do
         should contain_class('foreman::config::passenger').
-          with_listen_on_interface('').
+          with_ip(:undef).
```

Comment:
`nil` rather than :undef here, that's what it ends up evaluating to.  Most of the time, sigh.